### PR TITLE
Add gillchristian (myself 😅)

### PR DESCRIPTION
**Pull request by gillchristian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ A list of active declarative programming streams:
   - Speaking: EN
   - Programming Languages: **Haskell**, Purescript, Unison
   - Schedule: T/R 7:30-10:30 PM EST
+- [gillchristian](gillchristian):
```
Comment (chiroptical) [inline README.md:9]: The `[...]` should contain the twitch link to your channel

File: README.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ A list of active declarative programming streams:
...
+  - Speaking: EN
+  - Programming Languages: TypeScript (fp-ts), Haskell, Purescript, Reason, Rust
+  - Schedule: Thursdays 6:00 PM UTC ([gillchristian.yxz/stream](https://gillchristian.xyz))
```
Comment (chiroptical) [inline README.md:12]: Should the second link be https://gillchristian.xyz/stream?

**Comment by chiroptical** [comment]
I am going to hold on this for just a bit. I want to make sure there is no other feedback from the FP Slack

File: README.md (hunk 1 above)
Comment (gillchristian) [inline README.md:9]:
```suggestion
- [gillchristian](https://twitch.tv/gillchristian):
```

File: README.md (hunk 2 above)
Comment (gillchristian) [inline README.md:12]:
```suggestion
  - Schedule: Thursdays 6:00 PM UTC ([gillchristian.yxz/stream](https://gillchristian.xyz/stream))
```

File: README.md (hunk 1 above)
Comment (gillchristian) [reply README.md]: You are totally right about both comments. I made the PR from the phone so i missed a few things 😅 fixed now

**Comment by chiroptical** [comment]
LGTM. I'll merge it tomorrow unless someone makes any comments in FP Slack.

**Pull request merged**